index.js

```diff
@@ -145,28 +150,28 @@ function formatValue(value) {
     // we then remove the whole wrapping
     // otherwise, the element would be surrounded by quotes: <div a={{b: '<div />'}} />
     return '<__reactElementToJSXString__Wrapper__>' +
-      toJSXString({ReactElement: value, inline: true}) +
+      toJSXString({ReactElement: value, inline: true, options}) +
```

Comment:
we will also need to move `recurse` and `formatValue` so that they are able to call `toJSXString`, and then we will also need to move the functions that call those, and so on